.github/actions/prepare-ios-simulator/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Prepare iOS Simulator'
+inputs:
+  device:
+    description: 'The device name'
+    required: true
+  ios:
+    description: 'The iOS version'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: 
+        sudo mkdir -p /Library/Developer/CoreSimulator/Profiles/Runtimes
+        sudo ln -s /Applications/Xcode_12.4.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime /Library/Developer/CoreSimulator/Profiles/Runtimes/iOS\ ${{ inputs.ios }}.simruntime
+        xcrun simctl list runtimes
+        ios=${{ inputs.ios }}
+        ios_runtime=${ios//./-}
+        xcrun simctl create custom-test-device "${{ inputs.device }}" "com.apple.CoreSimulator.SimRuntime.iOS-$ios_runtime"
```

Comment:
This probably creates new device all the time, possibly leading to space issues  in some time... Are devices deleted somewhere in the pipeline??  
<img width="332" alt="Snímek obrazovky 2021-03-15 v 17 52 42" src="https://user-images.githubusercontent.com/17381941/111190324-4058c180-85b7-11eb-951a-330eb289b26b.png">


![Snímek obrazovky 2021-03-15 v 17 51 50](https://user-images.githubusercontent.com/17381941/111190180-215a2f80-85b7-11eb-9927-2cf8c469759c.png)

Reply:
What you can do is to `xcrun simctl delete custom-test-device` and `xcrun simctl delete unavailable` to cleanup. problem with `xcrun simctl delete custom-test-device`  is that it will delete only last one because there are more `custom-test-device` instances with different UDID(UUIDs?) :D Always get those wrong :D